### PR TITLE
don't explicitly set release and debug flags

### PR DESCRIPTION
**Pull request by ngoldbaum**
As suggested by @isuruf in https://github.com/conda-forge/netcdf4-feedstock/issues/61

### CODE REVIEW

**Comment by isuruf** [comment]
Can you bump build number?

File: recipe/build.sh [hunk 1]
```diff
@@ -66,8 +64,6 @@ cmake -DCMAKE_INSTALL_PREFIX=${PREFIX} \
       -DBUILD_UTILITIES=ON \
       -DENABLE_DOXYGEN=OFF \
       -DENABLE_LOGGING=ON \
-      -DCMAKE_C_FLAGS_RELEASE=${CFLAGS} \
-      -DCMAKE_C_FLAGS_DEBUG=${CFLAGS} \
```
Comment (ocefpaf) [inline recipe/build.sh]: I don't recall but I believe we had reasons to add those :-/

@mingwandroid may remember...

**Comment by ngoldbaum** [comment]
Argh, well fair enough. If you guys want to bring back those flags please be careful to ensure that the debug flags are *not* set in the build. To check, see the test script in conda-forge/netcdf4-feedstock#61. Unfortunately the uses of C++ assers in libnetcdf mean that if these flags *are* set in the release build then if libnetcdf triggers an assert, the python interpreter will crash.

**Comment by dopplershift** [comment]
Is there some way we could hit and check for an assert in the recipe tests?

**Comment by ocefpaf** [comment]
> Argh, well fair enough.

Don't worry. Looks like @mingwandroid also remove these from the AR builds based on https://github.com/conda-forge/libnetcdf-feedstock/pull/56#issuecomment-423854556 and I cannot even remember why I added that :smile:

**Comment by mingwandroid** [comment]
Hah did I? Ok then!

**Comment by ocefpaf** [comment]
> It wasn't picking up our compilation flags with this.

You said it was not picking up our compilation flags **with** this, right?
Sorry if I got it wrong.

**Pull request merged**